draft-ietf-mops-streaming-opcons.md

```diff
@@ -413,15 +413,41 @@ Here are a few common resolutions used for video content, with typical ranges of
 
-In more immersive applications, where limited user movement ("three degrees of freedom plus", or 3DoF+) or full user movement ("six degrees of freedom", or 6DoF) is allowed, the required bitrate grows even further. In this case, immersive content is typically referred to as volumetric media. One way to represent the volumetric media is to use point clouds, where streaming a single object may easily require a bitrate of 30 Mbps or higher. Refer to {{MPEGI}} and {{PCC}} for more details.  
+In more immersive applications, where limited user movement ("three degrees of freedom plus", or 3DoF+) or full user movement ("six degrees of freedom", or 6DoF) is allowed, the required bitrate grows even further. In this case, immersive content is typically referred to as volumetric media. One way to represent the volumetric media is to use point clouds, where streaming a single object may easily require a bitrate of 30 Mbps or higher. Refer to {{MPEGI}} and {{PCC}} for more details.
+
+## Path Bandwidth Constraints {#sec-band-constraints}
+
+Even when the bandwidth requirements for video streams along a path are well understood, additional analysis is required to understand the contraints on bandwidth at various points in the network. This is the case for two reasons:
+
+* First, media servers typically implement transport protocols probe for bandwidth, so when a media server's transport protocol implementation will detect a bottleneck link somewhere along the path to the media player. The media server transport protocol implementation may detect this because the sender does not receive acknowledgements from the media player's transport protocol implementation, meaning that some media packets have been lost, or because the diffence in spacing between acknowledgements is increasing, meaning that the media packets haven't been lost, but being stored in queues on a network element, and that these queues are growing because the network element is receiving media packets faster than they can be forwarded toward the media player. When the transport protocol implementation detects these conditions, it will typically reduce its "sending window", so that the media sender does not overwhelm the bottleneck link and cause congestion that potentially affects other media servers, or even the media server itself. Thia ia called "self congestion". These transport protocol-level responses are described in greater detail, in {{sec-trans}}.
```

Comment:
That last point that I mentioned I liked is the one that's sort of covered already in section 4.5 (https://htmlpreview.github.io/?https://github.com/ietf-wg-mops/draft-ietf-mops-streaming-opcons/blob/gh-pages/draft-ietf-mops-streaming-opcons.html#name-bitrate-detection-challenge), namely that there's 2 control loops involved that are not always doing the same thing.

Reply:
@GrumpyOldTroll, that comment is **super** helpful. I'm going to (gently?) refactor what I had in more detail in Section 2.2, WAY earlier in the document, between 2.2 and 4.5, and so that what's still in 2.2 actually belongs following 2.1.